src/containers/org/OrgDataSetContainer.js

```diff
@@ -126,7 +129,14 @@ const OrgDataSetContainer = ({
             <StackGrid>
               <SpaceBetweenGrid>
                 <div>
-                  <Typography variant="h1">{title || name}</Typography>
+                  <Typography gutterBottom variant="h1">{title || name}</Typography>
+                  <Typography variant="subtitle1">DATASET ID</Typography>
+                  <ActionsGrid align={{ v: 'center' }} fit>
+                    <Pre>{dataSetId}</Pre>
+                    <CopyButton
+                        aria-label="copy organization id"
+                        onClick={() => clipboardWriteText(dataSetId)} />
+                  </ActionsGrid>
```

Comment:
Can you move this below `SpaceBetweenGrid`? The reason is that the menu button is getting centered with the height of the title + this new stuff and it doesn't look as good, in my opinion.  Additionally, it will look consistent across pages (data set, org, role, member).

![Uploading 127042156-2111d33c-0aae-4bc4-9827-efe6aa7b1557.png…]()